### PR TITLE
adds champagne

### PR DESCRIPTION
**Pull request by Verslebas**
## About the PR

adds champagne to make the captains stop drinking vodka!

**Media**

ooh-champ!

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: Verslebas
- add: Added champagne!

### CODE REVIEW

**Comment by Verslebas** [comment]
bump

File: Resources/Locale/en-US/reagents/meta/consumable/drink/alcohol.ftl [hunk 1]
```diff
@@ -55,6 +55,9 @@ reagent-desc-whiskey = A type of distilled alcoholic beverage made from fermente
 reagent-name-wine = wine
 reagent-desc-wine = An premium alchoholic beverage made from distilled grape juice.
 
+reagent-name-champagne = champagne
+reagent-desc-champagne = An premium sparkling wine
```
Comment (PJB3005) [inline Resources/Locale/en-US/reagents/meta/consumable/drink/alcohol.ftl:59]: Shouldn't this be "A premium sparkling wine" instead?

File: Resources/Textures/Objects/Consumable/Drinks/absintheglass.rsi/meta.json [hunk 2]
```diff
@@ -1 +1 @@
-{"version": 1, "size": {"x": 32, "y": 32}, "license": "CC-BY-SA-3.0", "copyright": "https://github.com/discordia-space/CEV-Eris/raw/f7aa28fd4b4d0386c3393d829681ebca526f1d2d/icons/obj/drinks.dmi", "states": [{"name": "icon"}]}
\ No newline at end of file
+{"version": 1, "size": {"x": 32, "y": 32}, "license": "CC-BY-SA-3.0", "copyright": "https://github.com/discordia-space/CEV-Eris/raw/f7aa28fd4b4d0386c3393d829681ebca526f1d2d/icons/obj/drinks.dmi", "states": [{"name": "icon"}]}
```
Comment (PJB3005) [inline Resources/Textures/Objects/Consumable/Drinks/absintheglass.rsi/meta.json:1]: This file shouldn't be modified.

**Review by PJB3005 — CHANGES_REQUESTED** [review]

File: Resources/Textures/Objects/Consumable/Drinks/absintheglass.rsi/meta.json (hunk 2 above)
Comment (Verslebas) [reply Resources/Textures/Objects/Consumable/Drinks/absintheglass.rsi/meta.json:1]: > This file shouldn't be modified.

I accidentally changed this file, but then returned everything back

File: Resources/Locale/en-US/reagents/meta/consumable/drink/alcohol.ftl (hunk 1 above)
Comment (Verslebas) [reply Resources/Locale/en-US/reagents/meta/consumable/drink/alcohol.ftl:59]: > Shouldn't this be "A premium sparkling wine" instead?

Oops

Comment (Verslebas) [reply Resources/Locale/en-US/reagents/meta/consumable/drink/alcohol.ftl]: done

**Review by PJB3005 — APPROVED** [review]

**Pull request merged**